cli/dcoscli/data/help/marathon.txt

```diff
@@ -92,6 +95,15 @@ Commands:
         Display detailed information for a specific pod.
     pod update
         Update a pod.
+    debug list
+        Print a list of currently queued instance launches for
+        debugging purpose.
+    debug summary
+        Display summarized information for a queued instance launch
+        for debugging purpose.
+    debug details
+        Display detailed information for a queued instance launch
+        for debugging purpose.
```

Comment:
Have we thought about calling the command something else? The command should we named by what it does, not by what it could be used for (in this case debugging). Here are some suggestions for the name: `deployment` (I know this is already used but it feels like the best name), `pending`, `queue`, `staled`.

cc @pyronicide